src/Libraries/CoreNodeModels/CoreNodeModelsImages.resx

```diff
@@ -1343,6 +1343,39 @@
         PIw7vteQPuhLVMkA479rRIQ9hegBi3gVlTPYvr1bMcEHxT5Ginf+R2dm53gHaDPpF5En0WZeR3VH6qT1
         trlz75fwK2w6VIVFBQ7AJvsESZk6BO/iW/E+9u9A57xW7BNsVoaQqfvVwBky/FPkch8Yc7bTvUnhewAA
         AABJRU5ErkJggg==
+</value>
+  </data>
+  <data name="CoreNodeModels.Input.CustomSelectionNodeModel.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
I think this needs to be `CoreNodeModels.Input.CustomSelection` vs `CoreNodeModels.Input.CustomSelectionNodeModel`.  The icons are not loading